src/frontends/tensorflow_common/src/op/bias_add.cpp

```diff
@@ -41,14 +55,21 @@ OutputVector translate_bias_add_op(const NodeContext& node) {
         std::vector<int64_t> axes_unsqueeze;
         for (int64_t dim_ind = 0; dim_ind < value_rank; ++dim_ind) {
             if (dim_ind != 1) {
-                axes_unsqueeze.push_back(dim_ind);
+                axes_unsqueeze.push_back(dim_ind);  
             }
         }
         auto axes_unsqueeze_node =
             make_shared<v0::Constant>(element::i64, Shape{axes_unsqueeze.size()}, axes_unsqueeze);
         bias_reshaped = make_shared<v0::Unsqueeze>(bias, axes_unsqueeze_node);
     }
 
+    if (complex_type_inputs){
+        auto complex_add_res = make_shared<v1::Add>(value, bias_reshaped);
+        auto complex_result = make_shared<ComplexTypeMark>(complex_add_res->output(0), complex_type_mark_value->get_complex_part_type());
+        set_node_name(node.get_name(), complex_result);
```

Comment:
let us call `set_node_name` for `complex_add_res`. In this case you can simplify your code a bit, please take at the code below and check duplication:)